GTG/gtk/preferences.py

```diff
@@ -165,35 +156,22 @@ def valid_check(self, widget):
         self.refresh_hour.modify_fg(Gtk.StateFlags.NORMAL, color)
         self.refresh_mins.modify_fg(Gtk.StateFlags.NORMAL, color)
 
-    def on_close(self, widget, data=None):
-        """ Close the preferences dialog."""
-        self.valid_check(widget)
-        self.timer.time_changed()
-        self.dialog.hide()
-        return True
-
-    @classmethod
-    def on_help(cls, widget):
-        """ Open help for preferences """
-        help.show_help("preferences")
-        return True
-
     @classmethod
-    def on_autostart_toggled(cls, widget):
+    def on_autostart_toggled(cls, widget, state):
         """ Toggle GTG autostarting with the GNOME desktop """
         if widget.get_active():
             enable_gtg_autostart()
         else:
             disable_gtg_autostart()
 
-    def toggle_preview(self, widget):
+    def toggle_preview(self, widget, state):
         """ Toggle previews in the task view on or off."""
         curstate = self.config.get("contents_preview_enable")
         if curstate != widget.get_active():
```

Comment:
can you use `self.preview_button` ? It is how it is done everywhere else and it is more explicit.